war/src/main/less/pluginSetupWizard.less

```diff
@@ -544,12 +552,24 @@
 
         .dependent {
             color: #aaa;
-            padding-left: .4em;
+            padding-left: 10px;
 
             &:before {
                 content: ' ** ';
             }
         }
+        
+        .dependency-legend {
+            color: #aaa;
```

Comment:
When viewing the actual dialog, this is so difficult to notice it's basically invisible. Why not just make it black so it's more noticeable?